tools/armnn

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+root=$(cd $(dirname ${0})/..; pwd)
+build=${root}/build
+src=${root}/src
+third_party=${root}/third_party
+tools=${root}/tools
+
+identifier=armnn
+
+bold() {
+    echo "$(tty -s && tput bold)$1$(tty -s && tput sgr0)" 
+}
+
+git_sync () {
+    mkdir -p "${third_party}"
+    if [ -d "${third_party}/${1}" ]; then
+        git -C "${third_party}/${1}" pull --quiet --prune
+    else
+        git -C "${third_party}" clone --quiet --recursive ${2}
+    fi
+    git -C "${third_party}/${1}" submodule sync --quiet
+    git -C "${third_party}/${1}" submodule update --quiet --init --recursive
+}
+
+clean() {
+    bold "armnn clean"
+    rm -rf ${third_party}/armnn
+}
+
+sync() {
+    bold "armnn sync"
+    git_sync flatbuffers https://github.com/google/flatbuffers.git
+    git_sync armnn https://github.com/ARM-software/armnn.git
+}
+
+install() {
+    bold "armnn install"
+    case "$(uname)" in
+        "Linux")
+            [ -n "$(which cmake)" ] || sudo apt install -y cmake
+            ;;
+        "Darwin")
+            brew list cmake > /dev/null 2>&1 || brew install cmake > /dev/null
+            ;;
+    esac
+    pushd "${third_party}/flatbuffers" > /dev/null
+    cmake -G "Unix Makefiles" . > /dev/null
+    make > /dev/null
+    popd > /dev/null
+}
+
+schema() {
+    bold "armnn schema"
+
+    # master branch is patched not to make problem with js but current release branch is not patched yet
```

Comment:
Would it make sense to clone the `master` branch?

Reply:
fixed